src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -15,6 +15,51 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal partial class Binder
     {
+        internal struct NamespaceOrTypeOrAliasSymbolWithAnnotations
+        {
+            internal static NamespaceOrTypeOrAliasSymbolWithAnnotations CreateNonNull(bool nonNullTypes, Symbol symbol)
```

Comment:
>CreateNonNull [](start = 72, length = 13)

"CreateUnannotated"?

Reply:
Let's rename in a separate PR so that this PR is simply splitting the type hierarchy.

---
In reply to: [202119535](https://github.com/dotnet/roslyn/pull/28484#discussion_r202119535) [](ancestors = 202119535)